frontend/src/cljs/imas_seamap/views.cljs

```diff
@@ -714,6 +786,7 @@
       :isOpen   open?
       :onClose  #(re-frame/dispatch [:seamap-drawer/close])}
      [:div.seamap-drawer
+      {:key 1}
```

Comment:
I'm sure this is a todo, but noting here just in case (it's probably fine in this instance, it just looks like a code smell!  If these are panels, the name is probably fine as an id)

Reply:
Yeah, I'm not sure what's up with this, but it doesn't appear to be for the panels (happens for a drawer without the panels, so it's potentially related to the drawer?